src/InterNations/Sniffs/Syntax/MethodTypeHintsSniff.php

```diff
@@ -57,23 +57,29 @@ public function process(CodeSnifferFile $file, $stackPtr)
         $classPtr = $file->findPrevious(T_CLASS, $stackPtr);
         $className = $tokens[$file->findNext(T_WHITESPACE, $classPtr + 1, null, true)]['content'];
 
+        if (strpos($className, 'Sniff') !== false) {
```

Comment:
Yes, i will do that, unfortunately i have to skip sniff files, i can not have the param type hint and return type here because, it complains that definition should be the same as parent ones.